README.md

```diff
@@ -11,672 +11,292 @@
+_Availability and installation help_ of the
+[Release Tour](https://github.com/sagemath/sage/releases) for that version.
 
-More details, providing a background for these instructions, can be found
-in the section [Install from Source Code](https://doc.sagemath.org/html/en/installation/source.html)
-in the Installation Guide.
+Instead of your local platform, you may use online free platforms
 
+[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/sagemath/sage/tree/master
+) &nbsp; [![Open in GitHub Codespaces](https://img.shields.io/badge/Open_in_GitHub_Codespaces-black?logo=github)](https://codespaces.new/sagemath/sage/tree/master)
 
-1.  Decide on the source/build directory (`SAGE_ROOT`):
+that automatically clones the git repository for you. There you may build Sage from
+the source following the same instructions presented below.
 
-    - On personal computers, any subdirectory of your :envvar:`HOME`
-      directory should do.
+For examples in this guide, we assume Ubuntu Linux distribution for convenience.
 
-    - For example, you could use `SAGE_ROOT=~/sage/sage`, which we
-      will use as the running example below.
+### Preparing the Windows Platform
 
-    - You need at least 10 GB of free disk space.
+You may install Sage on Windows using Windows Subsystem for
+Linux (WSL). Follow the [official WSL guide](https://learn.microsoft.com/windows/wsl/)
+to install Ubuntu or other Linux distribution into WSL. Then open the Linux terminal.
+From this point on, all instructions for installation in Linux apply.
 
-    - The full path to the source directory must contain **no spaces**.
+### Preparing the Linux Platform
 
-    - After starting the build, you cannot move the source/build
-      directory without breaking things.
+Install the build prerequisites:
 
-    - You may want to avoid slow filesystems such as
-      [network file systems (NFS)](https://en.wikipedia.org/wiki/Network_File_System)
-      and the like.
+    sudo apt-get update
+    sudo apt-get install autoconf automake libtool pkg-config
 
-    - [macOS] macOS allows changing directories without using exact capitalization.
-      Beware of this convenience when compiling for macOS. Ignoring exact
-      capitalization when changing into :envvar:`SAGE_ROOT` can lead to build
-      errors for dependencies requiring exact capitalization in path names.
+For your platform, there is a similar list of
+[build prerequisites](https://doc.sagemath.org/html/en/reference/spkg/_bootstrap.html#equivalent-system-packages).
+Install them using the package manager of your platform.
 
-2.  Clone the sources with `git`:
+Sage depends on many system packages. Install them by executing
 
-    - To check that `git` is available, open a terminal and enter
-      the following command at the shell prompt (`$`):
+    sudo apt-get install bc binutils bzip2 ca-certificates cliquer cmake curl \
```

Comment:
Strong -1 on hardcoding this list here. 
It will be untested and therefore fall out of date.

Reply:
OK. Thanks.